### DISCUSSION TITLE
How to disable effect sharing (knobs) in mqtt (domoticz)?

### DISCUSSION THREAD

**Discussion by Barry40** [Koenkk/zigbee2mqtt#19260] [Devices]
Hi all, i'm a happy user of z2m, but i have a question:

Is it possible to only share (or enable) the switch (or dimmer) knob button instead of all other knobs like effect/do_not_disturb, etc in z2mqtt (of domoticz)?

So, in short words, is it possible to disable sharing knobs or only enable the knobs i need?

My domoticz is getting spammed with all other knobs wich i dont use, and only need the switch buttons, nothing else.

Thanks alot!
Barry

**Comment by wdonker** [comment]
Not a real solution maybe, but in Domoticz you can mark the unwanted devices as 'unused' (in the devices list). Then they won't show up on the other tabs anymore.

**Reply by Barry40** [reply]
I wish it was that simple, but after a restart they will be added again (mqtt autodiscover plugin is doing that)..
